chains/cheesechain/addresses.yaml

```diff
@@ -0,0 +1,11 @@
+staticMerkleRootMultisigIsmFactory: "0x8ca6f35F6134d7F26a82F1221bD813A0938485E0"
```

Comment:
These need to be sorted alphabetically. If you run `yarn lint:fix` it will fix it for you

Reply:
new contracts deployed by AW, can delete this addresses file